examples/gnrc_border_router/Makefile

```diff
@@ -53,5 +53,17 @@ CFLAGS += -DDEVELHELP
 
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
+TAP ?= tap0
+IPV6_PREFIX ?= 2001:db8::/64
 
 include $(RIOTBASE)/Makefile.include
+
+HOST_TOOLS := ethos uhcpd
+
+term: $(HOST_TOOLS)
+	sudo sh $(RIOTBASE)/dist/tools/ethos/start_network.sh $(PORT) $(TAP) $(IPV6_PREFIX)
+
+$(HOST_TOOLS): CC=gcc
+$(HOST_TOOLS): CFLAGS=
+$(HOST_TOOLS):
+	make -C $(RIOTBASE)/dist/tools/$@
```

Comment:
At least for me that resulted in `PATH` not being set => $(CC) was unable to work, but `env -u CC -u CFLAGS ...` works.